src/feed/lookup/linear.ts

```diff
@@ -0,0 +1,42 @@
+import Long from 'long'
+import { Epoch } from './epoch'
+import { Data } from '@ethersphere/bee-js/dist/src/types'
+
+export async function lookup(
+  time: Long,
+  hint: Epoch,
+  read: (epoch: Epoch, time: Long) => Promise<Data>,
+): Promise<Data> {
+  if (time.eqz()) {
+    time = Long.fromNumber(Math.round(Date.now() / 1000))
+  }
+
+  let previousChunk
+  let level = 31
+  for (;;) {
+    if (level <= 0) {
+      break
+    }
```

Comment:
why are you not using
```suggestion
  while (level > 0) {
```